src/app/policies/views/PolicyListView.vue

```diff
@@ -311,7 +319,7 @@ async function loadEntity({ name, mesh, path }: { name?: string | undefined, mes
 
 function changePolicyType(item: {value: string}) {
   router.push({
-    name: 'policy',
+    name: 'policy-list-view',
```

Comment:
Just something I keep noticing, not important here at all, but GH this doesn't syntax highlight this either 🤔